GettingStartedGuide.md

```diff
@@ -421,3 +452,158 @@ I (3444) coreMQTT: Packet received. ReceivedBytes=3.
 I (3444) ota_over_mqtt_demo: Subscribed to topic $aws/things/thing_esp32c3_nonOta/jobs/notify-next. 
 ```
 
+## 6 Run AWS IoT Qualification Test
+
+### 6.1 Prerequisite
+- Run [OTA](#5-perform-firmware-over-the-air-updates-with-aws-iot) once manually.
+- Enable "Run qualification test" by menuconfig (Featured FreeRTOS IoT Integration -> Run qualification test).
+- Enable Unity and Unity/Fixture by menuconfig.
+*Note: The log of module `esp_ota_ops` and `esp-tls-mbedtls` will be disabled when `Run qualification test` is on. You can enable them back by commenting out `esp_log_level_set` in [main.c](./main/main.c).
+
+### 6.2 Steps for each test case
+
+1. DEVICE_ADVISOR_TEST_ENABLED - device advisor test
+    - Set DEVICE_ADVISOR_TEST_ENABLED to 1 in [test_execution_config.h](./components/FreeRTOS-Libraries-Integration-Tests/config/test_execution_config.h).
+    - Create a device advisor test on website. ( Iot Console -> Test -> Device Advisor )
+    - Create test suite.
+    - Run test suite and set the device advisor endpoint to MQTT_SERVER_ENDPOINT in [test_param_config.h](./components/FreeRTOS-Libraries-Integration-Tests/config/test_param_config.h).
+    - Set MQTT_SERVER_PORT and IOT_THING_NAME (Same as provisioned one) in [test_param_config.h](./components/FreeRTOS-Libraries-Integration-Tests/config/test_param_config.h).
```

Comment:
I can't just use the configuration from menuconfig because IDT sets the configurations test_param_config.h during testing. We need to use the values in test_param_config.h for test. Just create a new method to set the configurations by menuconfig.